wire-tests/src/commonTest/proto-kotlin/com/squareup/wire/protos/custom_options/FooBar.kt

```diff
@@ -40,37 +40,53 @@ import kotlin.collections.List
 import okio.ByteString
 
 public class FooBar(
-  @MyFieldOptionOneOption(17)
```

Comment:
This is breaking then I think?

Reply:
I applied targets to property and field. I think the default is property, but I can remove the explicitness